app/src/androidTest/java/com/hedvig/app/feature/offer/OfferScreen.kt

```diff
@@ -36,9 +38,16 @@ class OfferScreen : Screen<OfferScreen>() {
     }
 }
 
-class ChangeDateSheet : Screen<ChangeDateSheet>() {
-    val autoSetDate = KButton { withId(R.id.auto_set_date_switch) }
-    val pickDate = KView { withId(R.id.date_pick_layout) }
+class ChangeDateBottomSheetScreen : Screen<ChangeDateBottomSheetScreen>() {
     val materialDatePicker = KMaterialDatePicker()
     val submit = KButton { withId(R.id.chooseDateButton) }
+    val changeDateContainer = KView {
+        withId(R.id.change_date_container)
+    }
+    val confirmSubmit = KAlertDialog()
+}
+
+class ChangeDateView : KScreen<ChangeDateView>() {
```

Comment:
```suggestion
object ChangeDateView : KScreen<ChangeDateView>() {
```

By making `KScreen`s `object`, we can write the DSL in this way:

```kotlin
ChangeDateView {
    someView { hasSomeProperty() }
}
```

instead of

```kotlin
onScreen<ChangeDateView> {
    someView { hasSomeProperty() }
}
```

Reply:
Additionally, this `KScreen` appears to be empty 🤔